### PR TITLE
L10 compatibility

### PR DESCRIPTION
**Pull request by saade**
Hi,

this PR adds L10 compatibility because its a dependency of Tenancy.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        laravel: [6, 8, 9]
+        php: ["7.3", "8.0", "8.1"]
+        laravel: ["^6.0", "^8.0", "^9.0", "^10.0"]
```
Comment (stancl) [inline .github/workflows/ci.yml]: We can drop PHP < 8 support and Laravel < 9 support (across all these PRs). Seems like Laravel < 9 doesn't receive security updates anymore, so we don't need to provide any further support. And L9 requires PHP 8.0, so 7.* isn't needed anymore.

This should simplify the workflow file, the listed|versions in `composer.json`, and make CI runs faster.

Comment (saade) [reply .github/workflows/ci.yml]: sweet! give me a couple minutes

**Comment by stancl** [comment]
See this:
- https://github.com/archtechx/jobpipeline/blob/fe0fc363e672a8b101997af628ccf838710ed191/src/JobPipeline.php#L96
- https://laravel.com/docs/10.x/upgrade#dispatch-now

Maybe we'll need a version check to determine whether `dispatch_now()` or `dispatch_sync()` should be used?

**Comment by saade** [comment]
We don't need to.  Laravel 9 already have the `dispatch_sync()` method implemented (since 8.x). We just need to adopt it instead of `dispatch_now()`

https://github.com/laravel/framework/blob/3bbc6c52477033e1f3d16ba84c673dac66e1b29c/src/Illuminate/Foundation/helpers.php#L404

**Comment by stancl** [comment]
Great, then dropping the old Laravel versions will simplify this as well.

**Comment by saade** [comment]
Updated all my PR's, let me know if i missed something

**Comment by stancl** [comment]
Seems like the L10 run is failing, it's probably because it's the only one using phpunit 10.

When we were working on updating other repos, we noticed that the phpunit.xml schema changed a bit. Not sure if that's the same issue as here. This might help: https://github.com/archtechx/virtualcolumn/pull/11/files#diff-ae310e922c85656813bd29d9f3c9a1b87c8aaa83d924813a5b91c67b7abb3cb9

**Comment by stancl** [comment]
Ah, just noticed that in that repo you updated the phpunit schema as well in your PR.

**Pull request merged**